applications/conf/routes

```diff
@@ -27,6 +27,9 @@ GET        /crosswords/accessible/$crosswordType<cryptic|quick|quiptic|quick-cry
 GET        /crosswords/search                                                                                                controllers.CrosswordSearchController.search()
 GET        /crosswords/lookup                                                                                                controllers.CrosswordSearchController.lookup()
 
+# Crosswords digital edition
+GET        /crosswords/digital-edition                                                                                        controllers.CrosswordEditionsController.digitalEdition
```

Comment:
Do we want to add this to the `dev-build` routes file?

Reply:
Good shout! Done @domlander